### ISSUE TITLE
Feature Request: Limit width of button

### ISSUE THREAD

**Issue by ccarns** [jrief/angular-bootstrap-plus#5]
If you choose more then a few options then the button can grow to large and wrap around but not show content. A suggested solution would be to replace the button content with X Selected after you reach a predetermined number (like >3)

**Comment by jrief** [comment]
Yes this would make sense. If you want to implement this, please send me a PR.
